packages/ember-codemod-remove-ember-css-modules/src/utils/blueprints/blueprints-root.ts

```diff
@@ -3,6 +3,8 @@ import { join } from 'node:path';
 import { getFilePath } from '@codemod-utils/blueprints';
 
 const fileURL = import.meta.url;
-const srcDirectory = join(getFilePath(fileURL), '../..');
 
-export const blueprintsRoot = join(srcDirectory, 'blueprints');
+export const blueprintsRoot = join(
+  getFilePath(fileURL),
+  '../../../../blueprints',
```

Comment:
**Technical risk**

The relative path shows `../../../..`, which is valid when the package is published, but is confusing for local development (we should have `../../..`). I'll need to investigate further how to publish a codemod written in TypeScript.